test/verify/check-machines

```diff
@@ -1459,15 +1459,6 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                                                                 os_name=config.FEDORA_28,
                                                                 os_short_id=config.FEDORA_28_SHORTID))
 
-            # Don't create root account
-            runner.createDownloadAnOSTest(TestMachines.VmDialog(self, sourceType='downloadOS',
```

Comment:
That also meant that this test doesn't actually validate the created user in the VM?

Reply:
It checks that the the parameters passed to virt-install are correct, but even when we pass `user-password-file` argument it will choose to create the user that has the same username with the one that runs the script, that is 'root'.
So it will create a root user with the password provided in the argument `user-password-file`, exactly as the bug report indicates.